schemas/questionnaire_v1.json

```diff
@@ -64,13 +64,7 @@
           },
           "type": {
             "type": "string",
-            "enum": [
-              "boolean",
-              "date",
-              "string",
-              "uuid",
-              "url"
-            ]
+            "enum": ["boolean", "date", "string", "uuid", "url"]
```

Comment:
The changes look good, but it looks like other reformatting has been done.

Reply:
https://github.com/ONSdigital/eq-schema-validator/pull/171

Reply:
https://github.com/ONSdigital/eq-schema-validator/pull/173

😁